### DISCUSSION TITLE
Running mongoDB local

### DISCUSSION THREAD

**Discussion by dominikjasek** [strapi/strapi#8476] [Legacy Topics]
Helo,

I have a question regarding approach to connecting database during development. Do you use connection to cloud database or localhost database mongodb? I am unable to connect to localhost, thu I am using cloud database.

In case you are using localhost database, how does your DATABASE_RUI key look like?

Thanks

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
